data/help/0-wifi.yaml

```diff
@@ -1,18 +0,0 @@
----
-key: wifi
-enabled: true
```

Comment:
I was debating turning it off vs removing it. You make a good point! I'll make the change you're suggesting about disabling this file instead of removing it.